src/index.html

```diff
@@ -7,12 +7,70 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <section
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__product-image"
+        src="./images/imac.jpeg"
+        alt="apple-imac-27"
+      />
+
+      <div class="product-card__product-details">
+        <h3 class="product-card__product-details-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
```

Comment:
The text inside the `h3` tag is quite long. To improve readability and maintainability, consider breaking it into multiple lines if it exceeds a certain length, as suggested in the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).